modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -73,22 +73,60 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					<aui:button cssClass="btn btn-primary btn-sm my-2" href="<%= cpContentHelper.getReplacementCommerceProductFriendlyURL(cpSku, themeDisplay) %>" value="replacement-product" />
 				</c:if>
 
-				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
-
-					<%
-					Format format = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
-					%>
+				<%
+				Format dateFormat = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
```

Comment:
can you change:

`Format dateFormat = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);`

to 

`Format dateFormatDateTime = FastDateFormatFactoryUtil.getDateTime(locale, timeZone);`

and also move this inside the 

`cpContentHelper.getOrderedIncomingQuantityLabel(commerceInventoryReplenishmentItems.get(0), dateFormat, request)`

then change it to:

`cpContentHelper.getOrderedIncomingQuantityLabel(commerceInventoryReplenishmentItems.get(0), request)`

Reply:
I can change the name of the variable. But I've made the format a parameter since the timezone cannot be fetch inside of that method, so passing the format would solve this problem. Also if in anytime we want to make the date format configurable, we will need only to change the jsp not the method.

Reply:
You can get the user from the request and the timezone from the user